src/System/PosixCompat/User.hsc

```diff
@@ -37,7 +40,78 @@ module System.PosixCompat.User (
 
 #include "HsUnixCompat.h"
 
+#ifdef UNIX_2_8
```

Comment:
Maybe just use `MIN_VERSION_unix(2, 8, 0)` here?

Reply:
Thank you for your review. I didn't know the `MIN_VERSION_*` macro. It seems that It's used at other points in this package. I'll use it. No need to use an extra flag if use it.